bescort.lic

```diff
@@ -420,12 +420,18 @@ class Bescort
   end
 
   def take_mammoth(mode)
-    if mode == 'fang'
-      manual_go2(2239)
-      mammoth_type = 'tall'
-    elsif mode == 'acen'
+    case mode
+    when 'fang'
+      id_to_mammoth = { 2239 => 'tall', 11130 => 'sea' }
+      closest_mammoth = sort_destinations([2239, 11130]).first
```

Comment:
What do you think about `closest_mammoth = sort_destinations(id_to_mammoth.keys).first`?